### DISCUSSION TITLE
Checking full node sync status

### DISCUSSION THREAD

**Discussion by whhpc19891120** [filecoin-project/lotus#7216] [Build and Sync]
after execute lotus-miner init  , the program stay here for half an hours, what is the reason?

2021-08-27T13:34:23.033+0800    INFO    main    lotus-miner/init.go:154 Trying to connect to full node RPC
2021-08-27T13:34:23.248+0800    INFO    main    lotus-miner/init.go:166 **Checking full node sync status**

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
the node is not synced

`lotus sync wait` should show it syncing

**Comment by whhpc19891120** [comment]
> the node is not synced
> 
> `lotus sync wait` should show it syncing

[root@ecs-d82f-0002 ~]# lotus sync status
sync status:
[root@ecs-d82f-0002 ~]# lotus sync wait

i execute the command it show nothing

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
thats a problem.

the daemon is running?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
@jennijuju this is a a discussion too. for sure not an issue with lotus

**Comment by JelloHuang** [comment]
i have tested with devnet .lotus sync wait show nothing.and genesis daemon show some thing below:
[genesis_daemon.md](https://github.com/filecoin-project/lotus/files/7072461/genesis_daemon.md)

**Comment by JelloHuang** [comment]
i am busy at sat and sunday . And tomorrow i will continue to  debug

**Comment by LexLuthr** [comment]
@whhpc19891120 Do you still need help with this problem?

**Comment by whhpc19891120** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好邮件已收到~查阅后会尽快回复~O(∩_∩)O谢谢~

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new input from the user for more than 7 days apart from the auto-reply mail above.
